tests/engine/models/test_actor_model.py

```diff
@@ -17,12 +18,45 @@ def test_actor_proper_shape():
     K = 3
     I = 100
 
+    return N, SE, BATCH_SIZE, max_N, UE, K, I
+
+
+def test_actor_proper_shape(params):
+    N, SE, BATCH_SIZE, max_N, UE, K, I = params
+
     example_user = torch.rand(BATCH_SIZE, UE)
     example_services_history = torch.rand(BATCH_SIZE, N, SE)
     example_mask = torch.ones(BATCH_SIZE, I)
-
     example_state = (example_user, example_services_history, example_mask)
 
     actor = Actor(K, SE, UE, I, MLPHistoryEmbedder(SE, max_N))
+    actor_output = actor(example_state)
+
+    assert (actor.act_min < actor_output).all()
+    assert (actor.act_max > actor_output).all()
```

Comment:
These assertions are tested in `test_actor_min_max`. Here we should test only the shape